src/UserWarning.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { Users } from './types/Users';
+import { addUser, getUser } from './api/users';
+
+interface Props {
+  setUser: React.Dispatch<any>;
```

Comment:
```suggestion
  setUser: React.Dispatch<React.SetStateAction<User | null>>;
```
it should be something like that type for useState setter function